src/SFA.DAS.EmployerFinance/Data/Scripts/Script0005CreateHealthChecks.sql

```diff
@@ -0,0 +1,12 @@
+﻿CREATE TABLE [dbo].[HealthChecks]
```

Comment:
How are these health checks being used as normally you wire this kind of check to App insights rather than store them in the database? I'm assuming you're not expecting high volumes of checks being made with the int ID?

Reply:
They’re manually triggered heath checks the same as in MA & Provider Relationships. It depends if you think 2 billion isn’t enough.

Reply:
The reason I wanted to check is that they are being linked to users so it depends how often these checks are being made and if they are aiming to be made for every user? Given the non-levy push is aiming to bring on potentially 1 million users or more this number may be reached depending on the answers to the questions above. You can always change it but then you expecting someone 4-5 years down the line on picking that up in time if it is a high rate check.

Reply:
They're manually triggered, over the next 4-5 years that means someone will have to create 1 million of them a day for it to be an issue. We're expecting at most 1 a day.